### ISSUE TITLE
Add imap and gd support failed

### ISSUE THREAD

**Issue by Judhianto** [devilbox/docker-php-fpm#90]
How to add php5-imap and php5-gd in php-fpm-5-3 ?

I have tried to add it using shell command in the container, but always failed.

Can you show me how to do that?

Thank you

**Comment by cytopia** [comment]
I will have a look, currently I am trying to add all modules that successfully install.
I give it another shot to add those.

**Comment by cytopia** [comment]
Just double checked, `gd` and `imap` are already included in PHP 5.3:

```bash
root@php-5.3.29 in /shared/httpd $ php -m | grep -iE 'gd|imap'
gd
imap
```
